src/components/button/button.api.ts

```diff
@@ -36,6 +36,10 @@ export interface IButtonBaseProps {
      * set (only-icon variant)
      */
     iconLeft?: IconType;
+    /**
+     * A boolean indicating whether the button is disabled.
+     */
+    disabled?: boolean;
```

Comment:
This isn't really necessary since we're extending the native HTML button and anchor elements.

Reply:
Good point!

Reply:
You forgot to remove this.